src/EditorFeatures/CSharpTest/Classification/TotalClassifierTests.cs

```diff
@@ -849,5 +849,89 @@ class Program : IReadOnlyCollection<int,string>
                 Punctuation.OpenCurly,
                 Punctuation.CloseCurly);
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
+        public async Task TestExtensionMethodDeclaration()
+        {
+            await TestAsync(
+@"static class ExtMethod
+{
+    public static void TestMethod(this C c)
+    {
+    }
+}
+",
+                Keyword("static"),
+                Keyword("class"),
+                Class("ExtMethod"),
+                Punctuation.OpenCurly,
+                Keyword("public"),
+                Keyword("static"),
+                Keyword("void"),
+                ExtensionMethod("TestMethod"),
+                Punctuation.OpenParen,
+                Keyword("this"),
+                Identifier("C"),
+                Parameter("c"),
+                Punctuation.CloseParen,
+                Punctuation.OpenCurly,
+                Punctuation.CloseCurly,
+                Punctuation.CloseCurly);
+        }
+
+       /* [Fact, Trait(Traits.Feature, Traits.Features.Classification)]
```

Comment:
Why the skip here? Can we get a proper Skip attribute applied if there's some tracking bug?